Makefile

```diff
@@ -1,13 +1,25 @@
-.PHONY: build
+GO_FILES?=$$(find . -name '*.go' |grep -v vendor)
 
+.PHONY: build
 build:
 	./build.sh
 
+.PHONY: build_redist
 build_redist:
 	./build_redist.sh
 
+.PHONY: build_samples
 build_samples:
 	./build_samples.sh
 
+.PHONY: local-fmt
+local-fmt:
+	gofmt -l -d $(GO_FILES)
+
+.PHONY: local-goimports
+local-goimports:
+	goimports -w $(GO_FILES)
```

Comment:
i like being able to format the code using a simple make target, not sure if this is something for openfaas org? I've found it annoying not to have 😄

Reply:
I believe that this happens automatically with VSCode, perhaps why it's not been an issue?